### ISSUE TITLE
generate creates empty passwords

### ISSUE THREAD

**Issue by doronbehar** [gopasspw/gopass#1026]
### Summary

`gopass generate test` creates empty password at first. Needs to be run again for a real password to be created there.

### Steps To Reproduce

1. `gopass generate test`
2. `gopass show -f test`

### Expected behavior

Random string.

### Actual behaviour

Empty line.

### Environment

- OS: Arch Linux
- OS version:
```
Linux NUC 4.19.12-arch1-1-ARCH #1 SMP PREEMPT Fri Dec 21 13:56:54 UTC 2018 x86_64 GNU/Linux
```
- gopass Version:
```
gopass 1.8.4 (a5442953608c3380efdddcfe2f6d6d123fd5605f) go1.11.4 linux amd64
<root>     - gpg 2.2.12 - git 2.20.1 -   fs 0.1.0
Available Crypto Backends: gpgcli, openpgp, plain, xc
Available RCS Backends: gitcli, noop
Available Storage Backends: fs, inmem
```
- Installation method: Arch Linux package from releases page.

### Additional context

Executable from official community package of Arch Linux repositories reports it is outdated. Bug [was reported](https://bugs.archlinux.org/task/61202) in Arch Linux's bug tracker.

**Comment by doronbehar** [comment]
Still the same when installed via `go get`.

**Comment by doronbehar** [comment]
- [My `config.yml`](https://github.com/doronbehar/.config_gopass/blob/master/config.yml).
- [Debug output of `gopass generate`](https://gist.github.com/doronbehar/233e9a29de65c8af9fc2c8b0e665b87c)
- [Debug output of `gopass show -f`](https://gist.github.com/doronbehar/8d02cb776dc47a9de24ef7a158a1b23f)

Could it be because my `.pass-template` is empty?

**Comment by dominikschulz** [comment]
Yes, indeed. That's most probably the cause. If a template exists, gopass will try to render it - no matter what content it has. If there are not statements producing any output, the secret will be empty.

There has been a change wrt. template handling in the last release, so maybe this was different before.

Maybe gopass should warn if the template is empty, but this would only cover a single failure mode. One could easily produce an non-empty template that doesn't generate any output either.

A compormise could be to print a notice if a template is being used to make it easier to debug.

**Comment by doronbehar** [comment]
This wasn't different in the previous release, only now I've found the time to report this. Thanks for your help!

**Comment by dominikschulz** [comment]
Good to hear!

I'm working on some small improvements to the UX. Let's leave this bug open to track that.
